content/java/concepts/calendar/terms/isSet/isSet.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.isSet()'
+Description: 'Checks whether the given calendar field has a value set or not.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.isSet(int calndr_field)`** method in `Calendar` class is used to check whether the given calendar field has a `value` set or not. All the `Cases` for which the values have been set by the calculations of the internal field are triggered by a `get` method call.
```

Comment:
```suggestion
The **`.isSet()`** method in `Calendar` class is used to check whether the given calendar field has a value set or not. All the cases for which the values have been set by the calculations of the internal field are triggered by a `get` method call.
```
The second sentence is not that clear. Can you rephrase it so it is more obvious what you're trying to say?

Reply:
Thank you for the feedback. I hope that it looks better with the new changes!